test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
http://git-scm.com/docs/git-blame

Authorship is already very well captured by git. Which issue/PR something is connected to has been annotated in github for a little while (less than a year though, so not all that long), but not as easy to capture from just local offline information.